src/main/java/training/supportbank/Printer.java

```diff
@@ -0,0 +1,25 @@
+package training.supportbank;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class Printer {
+    public static void listAll(String csvFile) throws IOException {
+        DataHandler dataHandler = new DataHandler();
+        BufferedReader reader = new BufferedReader(new FileReader(csvFile));
```

Comment:
This class is called `Printer`, but it has a lot of responsibilities, like reading the CSV and calling `calculateBalance`. What if it accepted a collection of objects to print, so it just has one responsibility?

I suspect this will help you when you get to the next part of the exercise, where you have to import a JSON file as well as a CSV.

Reply:
Done - Printer class now just iterates through input Arrays and prints to console. Other objects are instantiated in Main.